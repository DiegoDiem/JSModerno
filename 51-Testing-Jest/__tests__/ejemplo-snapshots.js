const cliente = {
    nombre:"Diego M",
    balance:500,
    tipo:"premium"
};

describe('TEsting al cliente',()=>{
    test('Es Diego Mendez',()=>{
        expect(cliente).toMatchSnapshot();
    })
})