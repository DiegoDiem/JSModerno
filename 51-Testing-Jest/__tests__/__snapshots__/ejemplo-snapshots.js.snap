// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEsting al cliente Es Diego Mendez 1`] = `
Object {
  "balance": 500,
  "nombre": "Diego M",
  "tipo": "premium",
}
`;
